README.md

```diff
@@ -11,16 +11,25 @@ AR2DTO (ActiveRecord to DTO, pronounced R2-D2 or Artoo-Detoo) is a gem that lets
 - [Installation](#installation)
 - [Usage](#usage)
   - [Setting up your models](#setting-up-your-models)
+  - [DTO class](#dto-class)
+  - [#to_dto](#to_dto)
+  - [.to_dto](#to_dto-1)
 - [Development](#development)
 - [Contributing](#contributing)
 - [License](#license)
 - [Code of Conduct](#code-of-conduct)
 
 ## Motivation
-TBD
+
+ActiveRecord is a very powerful tool and extensively used in most Rails apps. When working on large Rails apps, having such powerful objects all over the place can have a negative impact on the maintainability of the app. This is even more clear when trying to create domain boundaries within a Rails monolith. When there is communication between different components you probably don't want to share an ActiveRecord model with other components, if you do that, you'll be giving direct access to your component's tables from anywhere. For that reason, we want to create POROs that look like ActiveRecord models but that are much simpler and only carry data. This could be done by hand, but with this gem, we are trying to help you avoid having to write all the boilerplate to create these objects. As a corollary, by using this gem you are standardizing how things are done, and how your data-only objects look like.
 
 ### Why AR2DTO?
-TBD
+
+- It is a very small gem focused on solving one specific problem.
+- It helps you reduce boilerplate.
+- It provides a standard way to work with data-only objects.
+- It lets you work with objects that are similar to ActiveRecord models but without DB access or business logic.
+- It impedes ActiveRecord models leaking through its methods.
```

Comment:
WDYT about reordering these by importance?

I feel `It impedes ActiveRecord models leaking through its methods.` > `It is a very small gem focused on solving one specific problem.`